Lib/swigrun.swg

```diff
@@ -278,6 +278,7 @@ SWIG_TypeCheck(const char *c, swig_type_info *ty) {
       if (strcmp(iter->type->name, c) == 0) {
         if (iter == ty->cast)
           return iter;
+#ifndef Py_GIL_DISABLED
```

Comment:
I presume this is a performance optimisation you've disabled so that `SWIG_TypeCheck` can be called from multiple threads without having to add a lock ???

Reply:
Seems like it is having digested https://github.com/swig/swig/issues/103 more thoroughly.